$MirOS: ports/emulators/qemu/patches/patch-block-bochs_c,v 1.1 2005/05/03 19:30:03 tg Exp $
--- block-bochs.c.orig	Tue May  3 18:21:09 2005
+++ block-bochs.c	Tue May  3 18:21:12 2005
@@ -221,4 +221,8 @@ BlockDriver bdrv_bochs = {
     bochs_read,
     NULL,
     bochs_close,
+    NULL,
+    NULL,
+    NULL,
+    NULL
 };
